### PR TITLE
Add oc debug cmd information

### PR DESCRIPTION
**Pull request by ghost**
Trello card: https://trello.com/c/V1ieNYBR/837-document-oc-debug-in-either-the-admin-cli-or-developer-cli

### CODE REVIEW

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -658,6 +658,13 @@ Launch a command shell to debug a running application.
 $ oc debug [Options]
 ----
 
+For example, for the `oc debug <pod>|<object_type/object_name> [options] [-- command]` command:
+
+* `debug` is the action performed by the command.
+* `<object_type>` is the service or abbreviated svc the action is performed on.
+* `<object_name>` is the specified <object_type>.
+* `[options]` passes parameters to the program.
```
Comment (xingxingxia) [inline cli_reference/basic_cli_operations.adoc]: **\`<object_type>\` is the service or abbreviated svc
\`<object_name>\` is the specified <object_type>**
Hmm, the statement is not understandable.
To be clearer and simpler, now suggest the update just replace the line "oc debug [Options]" with "oc debug \<pod\> [options] [-- command]". I think this is OK enough

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

**Comment by mburke5678** [comment]
@tmorriso-rh LGTM

**Comment by xingxingxia** [comment]
LGTM, thank you

**Comment by tmorriso-rh** [comment]
@xingxingxia I have updated the PR based on recommendations from the OCP content strategist. PTAL.

@aheslin @sspeiche PTAL.

**Comment by xingxingxia** [comment]
The content LGTM

**Comment by sspeiche** [comment]
LGTM

**Comment by xingxingxia** [comment]
Hi, could this be merged now?

**Pull request merged**